homeassistant/components/zoneminder/__init__.py

```diff
@@ -26,20 +26,23 @@
 DEFAULT_VERIFY_SSL = True
 DOMAIN = 'zoneminder'
 
+HOST_CONFIG_SCHEMA = vol.Schema({
+    vol.Required(CONF_HOST): cv.string,
+    vol.Optional(CONF_PASSWORD): cv.string,
+    vol.Optional(CONF_PATH, default=DEFAULT_PATH): cv.string,
+    vol.Optional(CONF_PATH_ZMS, default=DEFAULT_PATH_ZMS): cv.string,
+    vol.Optional(CONF_SSL, default=DEFAULT_SSL): cv.boolean,
+    vol.Optional(CONF_USERNAME): cv.string,
+    vol.Optional(CONF_VERIFY_SSL, default=DEFAULT_VERIFY_SSL): cv.boolean,
+})
+
 CONFIG_SCHEMA = vol.Schema({
-    DOMAIN: vol.Schema({
-        vol.Required(CONF_HOST): cv.string,
-        vol.Optional(CONF_PASSWORD): cv.string,
-        vol.Optional(CONF_PATH, default=DEFAULT_PATH): cv.string,
-        vol.Optional(CONF_PATH_ZMS, default=DEFAULT_PATH_ZMS): cv.string,
-        vol.Optional(CONF_SSL, default=DEFAULT_SSL): cv.boolean,
-        vol.Optional(CONF_USERNAME): cv.string,
-        vol.Optional(CONF_VERIFY_SSL, default=DEFAULT_VERIFY_SSL): cv.boolean,
-    })
+    DOMAIN: cv.ensure_list([HOST_CONFIG_SCHEMA])
```

Comment:
Since we ensure a list, it's not a breaking change, correct?

Reply:
Yes, you're right. Removed from the breaking change description. This PR still causes a breaking change because of the service call schema change.

Reply:
Actually there's something wonky here. I'm not sure why this causing this behavior,

This works:
```
zoneminder:
  - host: zoneminder.dev.rohankapoor.com
```

This does not:
```
zoneminder:
  host: zoneminder.dev.rohankapoor.com
```